### PR TITLE
Improve visibility of documentation for pipeline visualisation on Databricks

### LINKED ISSUE #2285: Improve visibility of documentation for pipeline visualisation on Databricks
## Description
Improve the visibility of the part of the [documentation](https://kedro.readthedocs.io/en/stable/deployment/databricks.html#running-kedro-viz-on-databricks) for visualising pipelines on Databricks.

## Context
This issue is a child of https://github.com/kedro-org/kedro/issues/2185.

## Possible Implementation
The existing documentation could become a subsection of a new docs section dedicated to Databricks deployment. Adding images would give this section greater impact.

### PR DESCRIPTION
**Pull request by jmholzer**
## Description
Resolves #2285.

## Development notes
I added images to the documentation for pipeline visualisation, removed the references to Kedro-Viz in the title (as it may not be clear to users what it does) and made it clearer how to start load Kedro-Viz on Databricks.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/workflow_integration/databricks_visualisation.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 
 For Kedro-Viz to run with your Kedro project, you need to ensure that both the packages are installed in the same scope (notebook-scoped vs. cluster library). This means that if you `%pip install kedro` from inside your notebook then you should also `%pip install kedro-viz` from inside your notebook.
 If your cluster comes with Kedro installed on it as a library already then you should also add Kedro-Viz as a [cluster library](https://docs.microsoft.com/en-us/azure/databricks/libraries/cluster-libraries).
 
+To run Kedro-Viz on Databricks you must first [launch the Kedro IPython extension](./databricks_workspace.md#9-using-the-kedro-ipython-extension).
+
 Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line magic:
 
 ```ipython
 In [2]: %run_viz
```
Comment (yetudada) [inline docs/source/workflow_integration/databricks_visualisation.md:13]: Should we remove the `In [2]:` here?

**Review by yetudada — APPROVED** [review]
This is so cool to see! ❤️ I left minor comments.

File: docs/source/workflow_integration/databricks_visualisation.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 ```
+
+This command will present you with a link to the Kedro-Viz web application.
```
Comment (stichbury) [inline docs/source/workflow_integration/databricks_visualisation.md:16]:
```suggestion
This command presents you with a link to the Kedro-Viz web application.
```

File: docs/source/workflow_integration/databricks_visualisation.md (hunk 1 above)
Comment (stichbury) [reply docs/source/workflow_integration/databricks_visualisation.md:13]: I think so, either that or show it as a screen capture so it's clear that it comes from the notebook and is not something you type in.

File: docs/source/workflow_integration/databricks_visualisation.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
-# How to run Kedro-Viz on Databricks
+# Visualise a Kedro project in Databricks notebooks
 
-[Kedro-Viz](../visualisation/kedro-viz_visualisation.md) is a tool that allows you to visualise your Kedro pipeline. It is a standalone web application that runs on a web browser, it can be run on a local machine or on Databricks itself.
+[Kedro-Viz](../visualisation/kedro-viz_visualisation.md) is a tool that allows you to visualise your Kedro pipeline. It is a standalone web application that runs on a web browser, it can be run on a local machine or in Databricks notebooks.
```
[the remark this answers is not in the document]
Comment (stichbury) [reply docs/source/workflow_integration/databricks_visualisation.md:3]: Please can you avoid "allows you" and replace with "enables you" in most cases.

There's no "allow" -- it doesn't grant permission but offers the ability to do something, so typically it's a better choice to use "enable" or even, wince, "facilitates"

**Review by stichbury — APPROVED** [review]
Couple of minor changes but nothing significant. Thank you 🏆

**Review by astrojuanlu — APPROVED** [review]
Apart from the outstanding comments, LGTM!

**Pull request merged**